From 12f7ddf3d1e08e87881a71ca679c077d3538cbdc Mon Sep 17 00:00:00 2001
From: David S. Wang <dsw@cloudera.com>
Date: Wed, 27 Mar 2013 15:02:08 -0700
Subject: [PATCH 33/96] HBASE-7558 Client not able to access /hbase/unassigned in secure cluster

Reason: Bug
Author: Matteo Bertozzi
Ref: CDH-9101
(cherry picked from commit 07c90db23619bd0281d713f7b97752eb76afccb6)
---
 .../org/apache/hadoop/hbase/zookeeper/ZKUtil.java  |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/src/main/java/org/apache/hadoop/hbase/zookeeper/ZKUtil.java b/src/main/java/org/apache/hadoop/hbase/zookeeper/ZKUtil.java
index 2bc3e66..d6e1ba9 100644
--- a/src/main/java/org/apache/hadoop/hbase/zookeeper/ZKUtil.java
+++ b/src/main/java/org/apache/hadoop/hbase/zookeeper/ZKUtil.java
@@ -927,6 +927,7 @@ public class ZKUtil {
           (node.equals(zkw.clusterIdZNode) == true) ||
           (node.equals(zkw.rsZNode) == true) ||
           (node.equals(zkw.backupMasterAddressesZNode) == true) ||
+          (node.startsWith(zkw.assignmentZNode) == true) ||
           (node.startsWith(zkw.masterTableZNode) == true) ||
           (node.startsWith(zkw.masterTableZNode92) == true)) {
         return ZooKeeperWatcher.CREATOR_ALL_AND_WORLD_READABLE;
-- 
1.7.0.4

